.github/ISSUE_TEMPLATE/course-template.md

```diff
@@ -0,0 +1,44 @@
+---
+name: Course Template
+about: Describe this issue template's purpose here.
+title: Course - TUTORIAL TITLE
+labels: Awaiting Triage, [Content Type] Course
+assignees: ''
+
+---
+
+## Needs Assessment Statement
+<!-- A needs assessment statement is a summary of the gaps in knowledge and skills that the course will address. It helps identify the actual learning needs of the students, align course objectives with the desired outcomes, and ensure the right type of content is used to optimize learning effectiveness and engagement. -->
+
+## Target Audience
+<!- Describe the specific group of learners for whom the course is designed and intended. These learners usually share common characteristics, skill level, or professional goals, which enable the course creator to tailor the content and learning objectives to suit their needs. -->
+
+## Course Objective
+<!-- The course objective is a statement that describes the expected learning outcomes of the course. It is a concise statement that describes the knowledge and skills that the learners will acquire after completing the course. -->
+
+## Outline <a href="#outline" id="outline">#</a>
+<!-- The outline is a list of the topics that will be covered in the course. It is a high-level overview of the course content. Feel free to include links to exsiting content that will be used in the course. -->
+
+- [ ] https://learn.wordpress.org/tutorial/using-the-create-block-tool/
+- [ ] https://learn.wordpress.org/tutorial/converting-a-shortcode-into-a-block/
+- [ ] https://learn.wordpress.org/tutorial/styling-your-wordpress-block/
+- [ ] https://learn.wordpress.org/tutorial/using-block-attributes-to-enable-user-editing/
+
+# Related Resources
+Links to related content on Learn, HelpHub, DevHub, GitHub Gutenberg Issues, DevNotes, etc.
+- [ ]
+
+# Guidelines
+Review the [team guidelines](https://make.wordpress.org/training/handbook/guidelines/)
```

Comment:
Is this needed? Most of the links on the team guidelines page refer to Lesson Plans. Maybe the issue template should list the Course page in the handbook somewhere instead? https://make.wordpress.org/training/handbook/courses/

Reply:
I don't disagree with your point, however, the guidelines are still what's needed to be reviewed here, specifically the Brand Usage and Promotional Guidelines, and Bloom's taxonomy. 

What would be better is if that Resources page was updated to be more generalist, ie either move the Lesson Plan specific content to [Lesson Plans](https://make.wordpress.org/training/handbook/lesson-plans/) OR update the Resources page to include guidelines for all content, but that's a separate discussion.

Reply:
Good idea. I've made a note to update the "team guidelines" page so that it includes general info, and Lesson Plan specific content is moved. I'll update the handbook in the next few days. So, feel free to continue this link in.